Sources/Core/JavaIR.swift

```diff
@@ -26,16 +26,22 @@ struct JavaModifier: OptionSet {
     }
 }
 
+enum JavaAnnotation: String {
+    case override = "Override"
+    case nullable = "Nullable"
+    case nonnull = "NonNull"
+}
+
 public struct JavaIR {
     public struct Method {
-        let annotations: Set<String>
+        let annotations: Set<JavaAnnotation>
         let modifiers: JavaModifier
         let body: [String]
         let signature: String
 
         func render() -> [String] {
             // HACK: We should actually have an enum / optionset that we can check for abstract, static, ...
-            let annotationLines = annotations.map { "@\($0)" }
+            let annotationLines = annotations.map { "@\($0.rawValue)" }
```

Comment:
I think you'll need to sort the annotations to prevent the output from becoming non-deterministic

Reply:
The output stability test should fail if I'm right on this one

Reply:
Wait you didn't actually change that so if this was going to fail it would have already...